Libraries/LibWeb/Streams/AbstractOperations.cpp

```diff
@@ -2017,7 +2017,9 @@ GC::Ref<WebIDL::Promise> readable_stream_reader_generic_cancel(ReadableStreamGen
 // https://streams.spec.whatwg.org/#readable-stream-reader-generic-initialize
 void readable_stream_reader_generic_initialize(ReadableStreamReader reader, ReadableStream& stream)
 {
-    auto& realm = stream.realm();
+    // FIXME: Eactly when we should effectively be using the relevant realm of `this` is to be clarified by the spec.
```

Comment:
```suggestion
    // FIXME: Exactly when we should effectively be using the relevant realm of `this` is to be clarified by the spec.
```
Small typo here. Same as below.